lms/djangoapps/student_account/views.py

```diff
@@ -119,6 +119,14 @@ def login_and_registration_form(request, initial_mode="login"):
         'disable_footer': True,
     }
 
+    # Edraak: Check if the origin is a safe (trusted) url. if so pass it
+    # to the front end to allow navigating back to the originating
+    # site after authentication.
+    origin = request.GET.get('origin')
+    if settings.CORS_ORIGIN_ALLOW_ALL or \
+            origin in settings.CORS_ORIGIN_WHITELIST:
```

Comment:
@devalih this change will require that `programs.edraak.org` or `programs.edraakbeta.org` be listed in `settings.CORS_ORIGIN_WHITELIST` set by the playbooks. env file should look something like:
```
{
   ....
   "CORS_ORIGIN_WHITELIST": ['programs.edraak.org', 'www.edraak.org'],
   ....
}
```